notebooks/data/custom_des_trim/desogestrel/.bq_cache.csv.bb6fe5f2840a0b27d83ef15f2184227f.tmp

```diff
@@ -0,0 +1 @@
+File created by /root/.pyenv/versions/3.8.1/lib/python3.8/site-packages/ebmdatalab/bq.py
```

Comment:
These are the files that are created by `bq.cached_read`. They need to be there for the caching to work, otherwise the notebook tests will fail when it tries to do a query.

Might be worth thinking about calling them something other than `.tmp` files, as they'll always be needed when using BQ data in notebooks.

Reply:
Oh, sorry. I should have remembered that. That's a good example of why we should call them something different, I suppose!